pipeline/inputs/http.md

```diff
@@ -16,6 +16,10 @@ description: The HTTP input plugin allows you to send custom records to an HTTP
 | successful_response_code | It allows to set successful response code. `200`, `201` and `204` are supported.                                                              | 201     |
 | success_header           | Add an HTTP header key/value pair on success. Multiple headers can be set. Example: `X-Custom custom-answer`                                  |         |
 
+### TLS / SSL
+
+HTTP input plugin supports TTL/SSL, for more details about the properties available and general configuration, please refer to the [Transport Security](/administration/transport-security) section.
```

Comment:
Ah, it needs to be a relative link, e.g. https://github.com/fluent/fluent-bit-docs/blob/65df6a01e6ed32b03c9cb097c336b5543a86a458/pipeline/outputs/loki.md?plain=1#L180

Gitbook then does the magic I believe.